Android/res/layout/fragment_finish_geotagging.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@android:color/transparent"
+    android:fitsSystemWindows="true"
```

Comment:
@chaviw the container activity is not full screen or transparent, as such this fragment layout shouldn't need the `android:fitsSystemWindows` attribute.